remote_report_to_printer/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Report To Printer
+=================
+
+This module allows users to send reports to a printer using remote's configuration.
+
+A *remote* is an external entity that access odoo. For example, computers, mobiles,
+servers... We will identify *remotes* by its name or ip. On some environments,
+we want to configure the printers by remote, because users may change their
+work place.
+
+It adds an optional behaviour on reports to send it directly to a printer.
+
+* `Use remote's configuration` use the remote's behaviour
+
+Also, this option can be selected as a user default option for printer.
+
+Usage
+=====
+
+ * In *Settings > User & Companies > Remotes* configure the remotes printers by usage
```

Comment:
I think it's better to have their own menu entry that reuses "User & Companies".

Reply:
It depends on server-tools. I have created a PR there in order to change this: OCA/server-tools#1414. I will change the readme